maven-modules/maven-custom-plugin/counter-maven-plugin/pom.xml

```diff
@@ -55,15 +55,8 @@
         <plugins>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-plugin-plugin</artifactId>
-                <version>${maven-compiler-plugin.version}</version>
```

Comment:
Note that there was also previously a copy & paste error here: It erroneously used <code>maven-<b>compiler</b>-plugin.version</code>.